code/game/machinery/gulag_item_reclaimer.dm

```diff
@@ -47,8 +45,8 @@
 
 	var/obj/item/card/id/I = user.get_idcard(TRUE)
 	if(istype(I, /obj/item/card/id/prisoner))
-		var/obj/item/card/id/prisoner/P = I
-		if(P.points >= P.goal)
+		var/obj/item/card/id/prisoner/prisonerID = I
+		if(prisonerID.points >= prisonerID.goal && !prisonerID.permanent)
```

Comment:
I feel like this should have feedback reminding them that the ID is permanent.

Reply:
Well they shouldn't get to the reclaimer in the first place if their sentence is permanent but I could add a line to the UI, I guess.